packages/web-react/src/components/TextArea/useAdjustHeight.ts

```diff
@@ -0,0 +1,52 @@
+import { FormEvent, FormEventHandler, MutableRefObject, ForwardedRef } from 'react';
+
+export interface UseAdjustHeightProps {
+  elementReference?: MutableRefObject<ForwardedRef<HTMLTextAreaElement>>;
+  maxHeight: number;
+  onInput?: FormEventHandler<HTMLTextAreaElement>;
+  isAutoResizing?: boolean;
+}
+
+export interface UseAdjustHeightReturn {
+  onInput: FormEventHandler<HTMLTextAreaElement>;
+}
+
+export const useAdjustHeight = ({
+  elementReference,
+  maxHeight,
+  onInput,
+  isAutoResizing,
+}: UseAdjustHeightProps): UseAdjustHeightReturn => {
+  const adjustHeight = (textArea: HTMLTextAreaElement) => {
```

Comment:
In this case, a prefer using just `element` or `textareaElement` or just `textarea`. We are using `textArea` to distinguish from `textField`, e.g. `TextArea` and `TextField`. It sounds me just like that.

```suggestion
  const adjustHeight = (element: HTMLTextAreaElement) => {
```

Reply:
fixed